neo/Wallets/Wallet.cs

```diff
@@ -310,19 +314,24 @@ private Transaction MakeTransaction(Snapshot snapshot, TransactionAttribute[] at
                     if (script.IsSignatureContract())
                     {
                         size += 66 + script.GetVarSize();
-                        tx.NetworkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Neo_Crypto_CheckSig, null);
+                        tx.NetworkFee += priceBytes64 + priceBytes33 + priceCheckSig;
                     }
                     else if (script.IsMultiSigContract(out int m, out int n))
                     {
+                        long feeM, feeN;
+                        using (ScriptBuilder sb = new ScriptBuilder())
+                        {
+                            feeM = ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(m).ToArray()[0]];
+                            feeN = ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(n).ToArray()[1]];
```

Comment:
This is incorrect. Because `sb.EmitPush(m)` could have multi-bytes.

Reply:
It should be `X=BigInteger(m).ToArray().Size()` and the cost of equivalent `PUSHBYTESX`.